marshal

```diff
@@ -39,8 +39,8 @@ def main():
         'launch', help='Launch an image on a software simulator (defaults to qemu)')
     launch_parser.add_argument('-s', '--spike', action='store_true',
             help="Use the spike isa simulator instead of qemu")
-    launch_parser.add_argument('-j', '--job', nargs='?', default='all',
-            help="Launch the specified job. Defaults to running the base image.")
+    launch_parser.add_argument('-j', '--job', nargs='?', default=None, type=lambda s: [item for item in s.split(',')],
```

Comment:
It seems unlikely for a job name to intentionally include a comma, but if so, we could instead allow multiple `-j` options to accumulate  (e.g., `-j foo -j bar -j baz`):

```C
launch_parser.add_argument('-j', '--job', action='append')
```

Reply:
Ya, that's fair. It's also more clear that way, the list was somewhat of a hack and not really a standard thing. I've switched it to multiple -j options instead of a list. This required adding a new '--all' argument to specify launching all jobs.